github/client.go

```diff
@@ -420,50 +420,79 @@ func (client *Client) ForkRepository(project *Project) (repo *octokit.Repository
 	return
 }
 
-func (client *Client) Issues(project *Project) (issues []octokit.Issue, err error) {
-	url, err := octokit.RepoIssuesURL.Expand(octokit.M{"owner": project.Owner, "repo": project.Name})
+type IssueParams struct {
```

Comment:
>  Do you have an idea of what you would like to see here?

A link to https://developer.github.com/v3/issues/#create-an-issue that contains all the documentation for those fields and also list the other fields that we don't use (yet) in hub like assignee and milestone.